chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -627,7 +617,7 @@ def reset_cluster(self):
         self.reset_chroma_manager_db()
         self.remote_operations.stop_agents(s['address'] for s in config['lustre_servers'])
         if config.get('managed'):
-            self.remote_operations.clear_ha(config['lustre_servers'])
+            self.remote_operations.clear_ha(self.TEST_SERVERS)
```

Comment:
both config['lustre_servers'] and self.TEST_SERVERS refer to the lustre_servers in the config which are the storage servers. usage of the latter was more frequently used in this file hence the change for consistency